sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_day_6_v1/query.sql

```diff
@@ -65,15 +68,17 @@ SELECT
   attribution_source,
   distribution_id,
   attribution_ua,
-  COUNT(*) AS num_activated
+  COUNT(activated) AS num_activated,
+  COUNT(returned_second_day) AS returned_second_day,
+  COUNT(qualified_second_day) AS qualified_second_day,
+  COUNT(returned_week4) AS returned_week4,
+  COUNT(qualified_week4) AS qualified_week4,
 FROM
   client_conditions
 LEFT JOIN
   `moz-fx-data-shared-prod`.static.country_codes_v1 country_codes
 ON
   (country_codes.code = country_code)
-WHERE
-  activated = TRUE
```

Comment:
Is this going to be a problem now? Maybe we just need to re-work the final query that is powering the dashboard slightly.